ipaserver/install/httpinstance.py

```diff
@@ -263,30 +266,34 @@ def enable_mod_nss_renegotiate(self):
         installutils.set_directive(paths.HTTPD_NSS_CONF, 'NSSRenegotiation', 'on', False)
         installutils.set_directive(paths.HTTPD_NSS_CONF, 'NSSRequireSafeNegotiation', 'on', False)
 
-    def enable_mod_nss_ocsp(self):
+    def disable_mod_nss_ocsp(self):
+        if sysupgrade.get_upgrade_state('httpd', NSS_OCSP_ENABLED) is None:
+            self.__disable_mod_nss_ocsp()
+            sysupgrade.set_upgrade_state('httpd', NSS_OCSP_ENABLED, False)
```

Comment:
During testing I just noticed that in sysrestore Apache is referred to as 'httpd', but in sysupgrade it is referred to as 'http'. Can you please rename the section into 'http' in the sysupgrade checks?

Sorry that I did not notice it earlier, I again falled into a trap of assuming consistency in our code.